ckanext/restricted/templates/restricted/emails/restricted_user_allowed.txt

```diff
@@ -3,7 +3,7 @@
 {% trans %}You have requested access on {{ site_title }} to the resource: {{ resource_name }}.{% endtrans %}
 {% trans %}The contact person of the package has granted you access.{% endtrans %}
 
-{% trans %}Please click the following link: {{ resource_link }}{% endtrans %}
+{% trans %}Please click the following link{% endtrans %}: {{ resource_link|safe }}
```

Comment:
from what I know having variables like `{{ var_name }}` inside a translated string is totally ok and even helpful as it gives some context to the translator. I think we should keep it as it's been.
Good catch on the missing `|safe`. I think however that it still doesn't resolve the issue. The `resource_link` is an invalid ckan url and it doesn't do what it should. E.g. instead of:
https://adr.unaids.org/packages?id=vmmc-dmppt-2-data&esource_id=76c5b54c-4599-40aa-a65a-acaa8fbcd862
it should be:
https://adr.unaids.org/naomi-data-package/vmmc-dmppt-2-data/resource/76c5b54c-4599-40aa-a65a-acaa8fbcd862/edit

Reply:
> from what I know having variables like {{ var_name }} inside a translated string is totally ok and even helpful as it gives some context to the translator.

Sure, but not when the var is a url. It would turn `hello.com` into `witaj.com` which is not what we want.

> The resource_link is an invalid ckan url

I didn't test the link itself 🙈 
I was happy fixing the syntax and assuming that meant the link is ok, which isn't very smart haha
I'll look into that now. Thanks for the examples

Reply:
I've now fixed the url 👍